msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/BasicConfiguration.java

```diff
@@ -12,8 +12,9 @@
     String clientId;
     String authority;
     String redirectUri;
+    String logoutRedirectUri;
     String secretKey;
-    String oboApi;
+    String apiDefaultScope;
```

Comment:
Default is referring to the default nature of the scope. For example: api://a0f1b2d6-b573-44f4-8cb3-566933d2b71f/**.default**

I can see why this would be confusing though, maybe oboDefaultScope would be better here?

Reply:
oboDefaultScope probably a bit better